prebid-mobile/prebid-mobile-rendering.md

```diff
@@ -0,0 +1,110 @@
+
+- **Dispaly** - Banner/Interstitial
+- **Outsrteam Video** - Banner/Interstitial/Rewarded
+- **Native** - Disaply / Video
+
+Intefration steps are convinient and described in the respective docs for [iOS](TBD) and [Android](TBD). 
+
+## Motivation and Benefits 
+
+Rendering serves the general Prebid's aim - to be a trusted, transparent, and configurable monetization solution and gives a lot of benefits to both sides publishers and demand partners. 
+
+In comparising with no-rendering approach the new module gives next benefits:
+
+- Monitizaion without Ad Server. Publishers can monitize the apps through pure In-App bidding without additional ad requests.
+- Faster ad delivery. As soon as bid is recieved it can be displayed. It is applayed to both kind of integration with ad server  or without it.
+- Litle infrastructure. Since the rendering module doesn't need Prebid Cache it can be ommited. 
+- Less discrepancy, better analytics. Rendering module is able to track all needed events in the proper time according to the industry standard or demand side custom requrements. 
+- Full support of SKAdNetworks and similar frameworks 
+- MRAID 3.0 support
+- Better measurement. Ability to use any measurement provider. SDK implements the support of Open Measurement, the certification is scheduled to the upcoming releases.  
+- Open for improvements and adding exclusive features 
+- Support of Prebid Server distinct features like PG, omni-channel, floors, etc.  
+
+## How It Works
+
+The following diagrams show how the Rendering Module works. So far it supports three integration scenarious:
+
+- **Pure In-App** bidding without Primary Ad Server
+- **With Google** Ad Manger as a primary Ad Server
+- **With MoPub** as a primary ad Server
+
+### Pure In-App bidding
+
+![How Prebid Mobile Works]({{site.baseurl}}/assets/images/prebid-mobile/rendering/prebid-in-app-bidding-overview-pure-prebid.png){: .pb-lg-img :}
+
+1. Prebid SDK sends the bid request to the Prebid Server.
+1. Prebid Server runs the header bidding auction among preconfigured demand partners.
+1. Prebid Server responses with the winning bid.
+1. Prebid SDK renders the creative from recieved bid.
+
+### GAM Integration
+
+![How Prebid Mobile Works]({{site.baseurl}}/assets/images/prebid-mobile/rendering/prebid-in-app-bidding-overview-gam.png){: .pb-lg-img :}
+
+1. Prebid SDK makes a bid request. 
+1. Prebid Server runs the header bidding auction among preconfigured demand partners.
+1. Prebid SDK, via GAM Event Handler, sets up the targeting keywords into the GAM's ad request.
+1. GAM SDK makes an ad request and recieves the set of ad sources. 
+1. Basing on the creative the GAM Event Handler decides whether it's a Prebid Ad or no.
+1. The winner is displayed in the App with the respective rendering engine - prebid or GAM. GAM event handler manages the ad views.
+
+GAM Event Handlers determines that Prebid won basing on the next GAM features:
```

Comment:
GAM Event Handlers determines if a Prebid Line Item was selected in the ad server using the following GAM features: